README.md

```diff
@@ -154,13 +156,21 @@ optional arguments:
                         most frequent adapter in order for the former to be
                         considered as the library's 3'-end adapter (default: 2)
   --library-source-min-match-percentage FLOAT
-                        Minimum percentage of reads that are consistent with a
+                        minimum percentage of reads that are consistent with a
                         given source in order for it to be considered as the
                         to be considered the library's source.
   --library-source-min-frequency-ratio FLOAT
-                        Minimum frequency ratio between the first and second
+                        minimum frequency ratio between the first and second
                         most frequent source in order for the former to be
                         considered the library's source.
+  --library-type-max-distance INT
+                        upper limit on the difference in the reference sequence
+			coordinates between the two mates to be considered as
+			coming from a single fragment. (default: 1000)
+  --library-type-mates-cutoff FLOAT
+                        minimum fraction of mates that can be mapped to compatible
+			loci and are considered concordant pairs / all mates. 
+			(default: 0.95)
```

Comment:
Formatting seeems a little different than some other options (position and overall presence of `.` and the indentation.

However, I don't really care, as I think this should be removed from the `README.md` anyways. Not good to have this in too many places (docs, CLI args, models)...